slick.grid.js

```diff
@@ -3120,7 +3120,16 @@ if (typeof Slick === "undefined") {
       addlCssClasses += (formatterResult && formatterResult.addClasses ? (addlCssClasses ? ' ' : '') + formatterResult.addClasses : '');
       var toolTip = formatterResult && formatterResult.toolTip ? "title='" + formatterResult.toolTip + "'" : '';
 
-      stringArray.push("<div class='" + cellCss + (addlCssClasses ? ' ' + addlCssClasses : '') + "' " + toolTip + ">");
+      var customAttrStr = '';
+      if(m.cellAttrs) {
```

Comment:
can you add more type checks, I think it would be if you make sure this is defined and is an array before to looping over it.
```js
if (m.hasOwnProperty('cellAttrs') && m.cellAttrs instanceof Array) {
```